extensions/ccache-remote/README.server-setup.md

```
@@ -0,0 +1,141 @@
+   Or use a systemd unit that ties the announcement to `nginx` lifetime:
+   ```bash
+   cp misc/systemd/ccache-avahi-webdav.service /etc/systemd/system/
+   systemctl enable --now ccache-avahi-webdav
+   ```
+
+## DNS SRV Records (for remote/hosted servers)
+
+Set `CCACHE_REMOTE_DOMAIN` on the client, then create DNS records.
+
+Redis backend:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)